facefusion/face_helper.py

```diff
@@ -53,6 +53,30 @@ def paste_back(temp_frame : Frame, crop_frame : Frame, affine_matrix : Matrix) -
 	return temp_frame
 
 
+def paste_back(background: numpy.ndarray, foreground: numpy.ndarray, affine_matrix: numpy.ndarray, fade_amount: float = 0.25, offset_factor: Tuple[float, float, float, float] = ( 0.0, 0.0, 0.0, 0.0 )) -> Frame:
+    inverse_matrix = cv2.invertAffineTransform(affine_matrix)
+    f_height, f_width = foreground.shape[:2]
```

Comment:
no shortcuts in naming... this should be forground_height, forground_width... but of course to follow the existing code

```
temp_frame_height, temp_frame_width = temp_frame.shape[0:2]
crop_frame_height, crop_frame_width = crop_frame.shape[0:2]
```